### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by IrinaBabkina**
File with comments

### CODE REVIEW

File: Untitled.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# файл с программой не имеет нормального названия, говорящего о ее функции
```
Comment (agbragin) [inline Untitled.py]: Two concerns:
* By `comments` I mean GitHub comments (like this one and the one in the example)
* It's convenient to discuss programming-related things in English. Source code and comments should always be in English

Please, add another commit on top of this one with the content of the original file and provide comments in a form of code review comments.

File: Untitled.py [hunk 2]
```diff
@@ -1,55 +1,52 @@
-# файл с программой не имеет нормального названия, говорящего о ее функции
```
Comment (IrinaBabkina) [inline Untitled.py]: Uncorrect file name

File: Untitled.py [hunk 3]
```diff
@@ -1,55 +1,52 @@
...
-# лишние строки в начале кода
```
Comment (IrinaBabkina) [inline Untitled.py]: Extra empty lines

File: Untitled.py [hunk 4]
```diff
@@ -1,55 +1,52 @@
...
 
 for s in a.sheets():
-    if (s.name == 'Python') == True:  # лучше создать переменную с названием предмета. Условие должно быть записано через is True
+    if (s.name == 'Python') == True:
         out = sheet_data(s)
         if debug == 1:
             print(out)
 
 
-class PRESENT_RESULT  :  # создание класса без необходимости.
```
Comment (IrinaBabkina) [inline Untitled.py]: unreasonable class with an incorrect name (not CamelCase)

File: Untitled.py [hunk 5]
```diff
@@ -1,55 +1,52 @@
...
-    # Не "говорящее" имя класса, написанное не CamelCase
+class PRESENT_RESULT  :
     def __init__(self, a):
-        self.a=a         # нет пробелов вокруг равно.Нет пустой строки перед вторым def
```
Comment (IrinaBabkina) [inline Untitled.py]: there are no spaces around =
No empty line before second def

File: Untitled.py [hunk 6]
```diff
@@ -1,55 +1,52 @@
...
+        self.a=a
     def __str__(self):
         return self.a
 
 
-# много пустых строк после класса
```
Comment (IrinaBabkina) [inline Untitled.py]: Extra empty lines

File: Untitled.py [hunk 7]
```diff
@@ -1,55 +1,52 @@
...
+
 
 print("RESULT:")
-# лишняя пустая строка
```
Comment (IrinaBabkina) [inline Untitled.py]: extra emply lines

File: Untitled.py [hunk 8]
```diff
@@ -1,55 +1,52 @@
...
+
 resultat = ''
 
 for _ in range(1, len(out)):
-	resultat += str(out[_])  # использована табуляция, а не должна
```
Comment (IrinaBabkina) [inline Untitled.py]: Uncorrect tabulation use

File: Untitled.py [hunk 9]
```diff
@@ -1,55 +1,52 @@
...
+	resultat += str(out[_])
 
-print(PRESENT_RESULT(resultat))  # вывод лучше осуществлять в файл, а не в консоль
```
Comment (IrinaBabkina) [inline Untitled.py]: it is better to output to a file rather than to a console

File: Untitled.py (hunk 2 above)
Comment (agbragin) [reply Untitled.py]: I would not say that it's incorrect, technically it is, but agree that naming the script `Untitled` is a poor practice. Good point!

File: Untitled.py [hunk 10]
```diff
@@ -1,55 +1,52 @@
-# файл с программой не имеет нормального названия, говорящего о ее функции
-# лишние строки в начале кода
 
 
 def sheet_data(sheet):
 
-    out = list()  # лучше использовать скобочки
```
[the remark this answers is not in the document]
Comment (agbragin) [reply Untitled.py]: Could you provide an example of using `()` in this case?

File: Untitled.py [hunk 11]
```diff
@@ -1,55 +1,52 @@
...
-        average=sum(learner)  # нет пробелов вокруг равно
+        average=sum(learner)
         out.append([learner[1], average])
 
     return out
 
 def sum(learner):
     average = 0
-    for score in range(2, len(learner)) :  # лишний пробел после двоеточия
-        average = average+learner[score] // 2  # нет пробелов вокруг +
```
[the remark this answers is not in the document]
Comment (agbragin) [reply Untitled.py]: Actually, there *are* spaces, so, please, add clarification of your expectations regarding spaces here.

File: Untitled.py [hunk 12]
```diff
@@ -1,55 +1,52 @@
...
-        if debug>0:                     # нет пробелов вокруг >
+    for score in range(2, len(learner)) :
+        average = average+learner[score] // 2
+        if debug>0:
             print(learner[score], average)
     return average
 
-import xlrd  # импорт должен быть в начале программы
-            # нет указания на необходимость подгрузки библиотеки
```
[the remark this answers is not in the document]
Comment (agbragin) [reply Untitled.py]: I would suggest the term `dependency` here.

File: Untitled.py [hunk 13]
```diff
@@ -1,55 +1,52 @@
...
+import xlrd
+
 a = xlrd.open_workbook("БИОЛОГИ_ таблица успеваемости осень 2018.xlsx")
```
[the remark this answers is not in the document]
Comment (agbragin) [reply Untitled.py]: All variables in Python are modifiable (obviously, not all of them are at the beginning of the file). Please, add clarification.

File: Untitled.py (hunk 8 above)
Comment (agbragin) [reply Untitled.py]: I would name it `mixed tabs and spaces`.

File: Untitled.py (hunk 9 above)
Comment (agbragin) [reply Untitled.py]: It depends on a task. Often it's better to pump the result into [standard output](https://en.wikipedia.org/wiki/Standard_streams#Standard_output_(stdout)) to let the user decide where to send the data (this is what `print` do as a default).

**Comment by agbragin** [comment]
I've marked all accepted comments as `resolved` and added questions to the ones I'm not sure about. Please, add comments where appropriate.

File: Untitled.py (hunk 11 above)
[the remark this answers is not in the document]
Comment (IrinaBabkina) [reply Untitled.py]: Spaces around +
Extra space before :

File: Untitled.py (hunk 13 above)
[the remark this answers is not in the document]
Comment (IrinaBabkina) [reply Untitled.py]: I want to say that variable with file name which user may want to change should be in first part of code

File: Untitled.py (hunk 9 above)
Comment (IrinaBabkina) [reply Untitled.py]: Ok

**Comment by IrinaBabkina** [comment]
I reviewed all open comments and clarified them. Thanks!

File: Untitled.py (hunk 13 above)
[the remark this answers is not in the document]
Comment (agbragin) [reply Untitled.py]: Currently, there is no variable for the name fo the file, but I suppose I've got your point.

File: Untitled.py (hunk 10 above)
[the remark this answers is not in the document]
Comment (IrinaBabkina) [reply Untitled.py]: Sorry, I think about [ ]

**Pull request merged**